node/silkworm/etl/collector.hpp

```diff
@@ -57,10 +56,11 @@ class Collector {
     void load(mdbx::cursor& target, LoadFunc load_func = nullptr,
               MDBX_put_flags_t flags = MDBX_put_flags_t::MDBX_UPSERT, uint32_t log_every_percent = 100u);
 
+    void load_serialized(mdbx::cursor& target, LoadFunc load_func = nullptr,
```

Comment:
What's the difference between `load` and `load_serialized`?

Reply:
Load serialized consumes one flushed after the other (wuthout passing from priority queue)
Normal load instead pulls an item from each flushed file and picks the lowest key to be processed

Reply:
Anyway this issue, imo, is very low priority and I would not tamper too much with etl load for the moment